organization.go

```diff
@@ -1,25 +1,117 @@
 }
 
 type Organization struct {
-	URL                *string           `json:"url,omitempty"`
-	ID                 *int              `json:"id,omitempty"`
-	Name               *string           `json:"name,omitempty"`
-	SharedTickets      *bool             `json:"shared_tickets,omitempty"`
-	SharedComments     *bool             `json:"shared_comments,omitempty"`
-	ExternalID         *string           `json:"external_id,omitempty"`
-	CreatedAt          *time.Time        `json:"created_at,omitempty"`
-	UpdatedAt          *time.Time        `json:"updated_at,omitempty"`
+	URL                string           `json:"url,omitempty"`
+	ID                 int              `json:"id,omitempty"`
+	Name               string           `json:"name,omitempty"`
+	SharedTickets      bool             `json:"shared_tickets,omitempty"`
+	SharedComments     bool             `json:"shared_comments,omitempty"`
+	ExternalID         string           `json:"external_id,omitempty"`
+	CreatedAt          string           `json:"created_at,omitempty"`
+	UpdatedAt          string           `json:"updated_at,omitempty"`
 	DomainNames        []string          `json:"domain_names,omitempty"`
-	Details            *string           `json:"details,omitempty"`
-	Notes              *string           `json:"notes,omitempty"`
-	GroupID            *string           `json:"group_id,omitempty"`
+	Details            string           `json:"details,omitempty"`
+	Notes              string           `json:"notes,omitempty"`
+	GroupID            string           `json:"group_id,omitempty"`
 	Tags               []string          `json:"tags,omitempty"`
 	OrganizationFields map[string]string `json:"organization_fields,omitempty"`
 }
+
+// TicketService struct
+type OrganizationService struct {
+	client *Client
+}
+
+// finds an organization in zendesk byt id
+func (s *OrganizationService) GetOrganizationById(organization_id string) (*Organization, *Response, error) {
+	org := OrganizationResponse{}
+
+	url := fmt.Sprintf("organizations/%s.json", organization_id)
+
+	req, err := s.client.NewRequest("GET", url, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	resp, err := s.client.Do(req, &org)
+	if err != nil {
+		return nil, nil,  err
+	}
+
+	return org.Organization, resp, err
+}
+
+//// updates and organization by id
+func (s *OrganizationService) UpdateOrganizationById(org *Organization) (*Organization, error) {
+	var organization *Organization
+	var err error
+
+	fmt.Println(org)
+
+	url := fmt.Sprintf("organizations/%v.json", org.ID)
+
+	or := &OrganizationWrapper{Organization: org}
+
+	fmt.Println(or.Organization.Notes)
+
+	req, err := s.client.NewRequest("PUT", url, or)
+	if err != nil {
+		return organization, err
+	}
+
+	result := new(OrganizationWrapper)
+
+	_, err = s.client.Do(req, result)
+	if err != nil {
+		return organization, err
+	}
+
+	organization = result.Organization
+
+	return organization, err
+}
+
+//creates a new organization
```

Comment:
comment on exported method OrganizationService.CreateOrganization should be of the form "CreateOrganization ..."